specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/examples/RoutingIntentDelete.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "routingIntentName": "Intent1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-03-01",
```

Comment:
Please update the API version in examples

Reply:
done